### PR TITLE
Run node security platform tool in CI

### LINKED ISSUE #1102: Run the node security platform tool in CI to check dependencies
It would be helpful to run the [nsp](https://github.com/nodesecurity/nsp) command as part of our continuous integration checks so that we can be sure we're never shipping any known vulnerabilities in the new AMO frontend.

This should be as simple as adding `npm check` to our `.travis.yml`.

So far so good:

```
$ nsp check
(+) No known vulnerabilities found
```

### PR DESCRIPTION
**Pull request by willdurand**
Fixes #1102

---

This PR add [nsp](https://github.com/nodesecurity/nsp) to detect dependencies with known vulnerabilities.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -33,5 +34,5 @@ script:
     yarn start-func-test-server;
   fi
 - if [ $TOXENV ]; then tox; fi
-- if [ $TASK ]; then yarn $TASK; fi
+- if [ $TASK ]; then yarn run $TASK; fi
```
Comment (tofumatt) [inline .travis.yml]: I don't think `yarn run` is needed; won't `yarn test` and `yarn run test` do the same thing?

File: README.md [hunk 2]
```diff
@@ -47,6 +47,7 @@ Here are some commands you can run:
 | yarn amo:dev                | Start the dev server/proxy (for amo) using data from the dev server (https://addons-dev.allizom.org/) |
 | yarn amo:no-proxy           | Start the dev server without a proxy (for amo) using data from Docker |
 | yarn amo:stage              | Start the dev server/proxy (for amo) using data from the staging server (https://addons.allizom.org/) |
+| yarn check                  | Run [nsp][] to check dependencies |
```
Comment (tofumatt) [inline README.md]: This should have a more descriptive name, so `nsp-check` or something. `check` is a bit too generic.

**Comment by willdurand** [comment]
It fails because it seems I must use a version with `^` in `package.json`.

File: .travis.yml (hunk 1 above)
Comment (willdurand) [inline .travis.yml]: In principle, unless `yarn` has an existing command with the same name.

Comment (tofumatt) [inline .travis.yml]: Right. I guess being more explicit on the CI is fine. 😄

**Comment by tofumatt** [comment]
Ah yes, for our `devDependencies` we use a `^` to prevent Greenkeeper PRs on every minor update.

**Comment by willdurand** [comment]
Still no known vulnerabilities: https://travis-ci.org/mozilla/addons-frontend/jobs/264733282 :bowtie:

**Comment by willdurand** [comment]
@tofumatt updated everything :)

**Comment by tofumatt** [comment]
Sweet, looks good! 👍

**Pull request merged**